### PR TITLE
Add Memos loader

### PR DESCRIPTION
**Pull request by p3psi-boo**
For [Memos](https://github.com/usememos/memos).

### CODE REVIEW

File: loader_hub/memos/base.py [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    """Memos reader.
+
+    Reads content from an Memos.
+
+    """
+
+    def __init__(self, host: str = "https://demo.usememos.com/") -> None:
+        """Init params."""
+        
+        self._memoUrl = host + "api/memo"
```
Comment (emptycrown) [inline loader_hub/memos/base.py:19]: Can we use `urljoin` here? It accounts for extra/missing slashes: https://docs.python.org/3/library/urllib.parse.html

File: loader_hub/memos/base.py [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+        """
+        import requests
+        
+        documents = []
+
+        realUrl = self._memoUrl
+        if ~params:
+            params = {}
+            realUrl += "/all"
```
Comment (emptycrown) [inline loader_hub/memos/base.py:38]: Same here: `urljoin` is best.

File: loader_hub/memos/base.py [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+        req = requests.get(realUrl, params)
+        
+        if ~req.ok:
```
Comment (emptycrown) [inline loader_hub/memos/base.py:42]: It's probably better to raise an error here instead of returning an empty list

**Review by emptycrown — CHANGES_REQUESTED** [review]
Thanks for contributing this! Left a couple of comments.

**Comment by p3psi-boo** [comment]
Fixed it.
Thanks for your suggestions.

**Comment by p3psi-boo** [comment]
Wait a min

File: loader_hub/memos/base.py [hunk 4]
```diff
@@ -31,21 +32,16 @@ def load_data(self, params: Optional[Any] = 101) -> List[Document]:
...
         realUrl = self._memoUrl
+        
         if ~params:
             params = {}
-            realUrl += "/all"
+            realUrl = urljoin(self._memoUrl, "/all")
 
         req = requests.get(realUrl, params)
-        
-        if ~req.ok:
```
Comment (emptycrown) [inline loader_hub/memos/base.py]: Err I meant if request is not ok, `raise ValueError("Your Memo URL is not valid")` or something like that

**Review by emptycrown — CHANGES_REQUESTED** [review]

File: loader_hub/memos/base.py (hunk 4 above)
Comment (p3psi-boo) [reply loader_hub/memos/base.py]: Ok

**Review by emptycrown — APPROVED** [review]

**Review by emptycrown — APPROVED** [review]

**Pull request merged**